content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -304,8 +304,13 @@ For more information, see the [Collector Health Metrics][8] documentation.
 To install or upgrade the Datadog Agent with OpenTelemetry Collector in your Kubernetes environment, use the following Helm command:
 
 ```shell
-helm upgrade -i <RELEASE_NAME> datadog/datadog -f datadog-values.yaml
+helm upgrade -i <RELEASE_NAME> datadog/datadog \
+  -f datadog-values.yaml \
+  --set-file datadog.otelCollector.config=otel-config.yaml
```

Comment:
Don't replace it. Add it as additional command, in case customers want to bring their own collector config

Reply:
Ah, gotcha. Fixed! Let me know if you have any other feedback.